src/inference/src/threading/ie_istreams_executor.cpp

```diff
@@ -197,6 +214,71 @@ void IStreamsExecutor::Config::SetConfig(const std::string& key, const std::stri
                        << ". Expected only non negative numbers (#threads)";
         }
         _threadsPerStream = val_i;
+    } else if (key == CONFIG_KEY(BIG_CORE_STREAMS)) {
+        int val_i;
+        try {
+            val_i = std::stoi(value);
+        } catch (const std::exception&) {
+            IE_THROW() << "Wrong value for property key " << CONFIG_KEY(BIG_CORE_STREAMS)
+                       << ". Expected only non negative numbers (#streams)";
+        }
+        if (val_i < 0) {
+            IE_THROW() << "Wrong value for property key " << CONFIG_KEY(BIG_CORE_STREAMS)
+                       << ". Expected only non negative numbers (#streams)";
+        }
+        _big_core_streams = val_i;
+    } else if (key == CONFIG_KEY(SMALL_CORE_STREAMS)) {
+        int val_i;
+        try {
+            val_i = std::stoi(value);
+        } catch (const std::exception&) {
+            IE_THROW() << "Wrong value for property key " << CONFIG_KEY(SMALL_CORE_STREAMS)
+                       << ". Expected only non negative numbers (#streams)";
+        }
+        if (val_i < 0) {
+            IE_THROW() << "Wrong value for property key " << CONFIG_KEY(SMALL_CORE_STREAMS)
```

Comment:
Please do not use same error wording in different errors.

Reply:
Ok, done.